### DISCUSSION TITLE
blog/2022/precommit-autoupdate/

### DISCUSSION THREAD

**Discussion by giscus** [choldgraf/choldgraf.github.io#70] [Blog comments]
# blog/2022/precommit-autoupdate/

I figured out a way to automatically update all of the git pre-commit hook versions at once! pre-commit is a useful command line tool for running simple commands before every git commit. I use it t...

https://chrisholdgraf.com/blog/2022/precommit-autoupdate/

**Comment by sudharsan2020** [comment]
This was really helpful :)

**Comment by dmanthing** [comment]
or use https://pypi.org/project/pre-commit-update/ and configure it for your needs
